keras_nlp/src/models/csp_darknet/csp_darknet_backbone.py

```diff
@@ -0,0 +1,408 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import keras
+from keras import layers
+
+from keras_nlp.src.api_export import keras_nlp_export
+from keras_nlp.src.models.backbone import Backbone
+
+
+@keras_nlp_export("keras_nlp.models.CSPDarkNetBackbone")
+class CSPDarkNetBackbone(Backbone):
+    """
```

Comment:
one liner should be on the same line as the triple quote. https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
Moved